x/ibc/03-connection/cli.go

```diff
@@ -0,0 +1,69 @@
+package connection
+
+import (
+	"bytes"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+
+	"github.com/cosmos/cosmos-sdk/x/ibc/23-commitment/merkle"
+)
+
+func (man Manager) CLIObject(connid, clientid string) Object {
```

Comment:
What is a "CLIObject" ? It doesn't seem to be a "command line interface object". "ClientObject" perhaps?